doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -384,91 +381,104 @@ Rewriting with definitional equality
            exact H.
          Qed.
 
-.. _performingcomputations:
+.. _applyingconversionrules:
 
-Performing computations
----------------------------
+Applying conversion rules
+-------------------------
+
+These tactics apply reductions and expansions, replacing :term:`convertible` terms
+with equivalent terms.
+They implement different specialized uses of the
+:tacn:`change` tactic.  Other ways to apply these reductions are through
+the :cmd:`Eval` command, the `Eval` clause in the :cmd:`Definition`/:cmd:`Example`
+command and the :tacn:`eval` tactic.
+
+Conversions (except, in some cases, WHICH ARE? for :tacn:`vm_compute` and
+:tacn:`native_compute`) only change existential variables
```

Comment:
Perhaps you want to say "only change the types of existential variables"?

Reply:
Because there are no "values", right?

The type and the context the same thing here?

![image](https://user-images.githubusercontent.com/1253341/107292511-d466c480-6a1e-11eb-9bb2-3d6f0025b833.png)

`Existential 1 = ?Goal : [ |- 2 + 3 + 4 = 0]`

Reply:
To change the value of an existential variable is to instantiate the existential with some term

I guess I mean "type and context" (in the paragraph you quote, "context" is things between `[` and `|-` and "type" is things between `|-` and `]`)